keps/sig-scheduling/20190131-pod-priority-preemption.md

```diff
@@ -11,7 +11,7 @@ approvers:
   - "@liggitt"
 editor: Babak Salamat
 creation-date: 2019-01-31
-last-updated: 2017-01-31
+last-updated: 2019-01-31
 status: impementable
```

Comment:
Now that you are at it, could you please fix the typo here: `implementable`

Reply:
Didn't catch that... Done.